service_container/factories.rst

```diff
@@ -100,6 +100,81 @@ create its object:
     the configured class name may be used by compiler passes and therefore
     should be set to a sensible value.
 
+You can also use the ``factory`` option to create services from static methods.
```

Comment:
this is what the previous example does already

seeing the link you put in the description, I was expecting to read something about it - aka that the class can be omitted if the static factory method is on the same class than the created instance